src/components/Peace/Tab/Point.tsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/jsx-pascal-case */
+import COLOR from "constants/color";
+import { FlexBox, FlexTextBox } from "components/common";
+import { styled as style } from "@mui/material/styles";
+import styled from "styled-components";
+import * as TooltipLib from "@mui/material/Tooltip";
+import { TooltipProps, tooltipClasses } from "@mui/material/Tooltip";
+
+const dummyPoint = [
+  {
+    title: "오늘은 쓰레기를 한 개만 버려보자",
+    isPlus: true,
+    point: 10,
+    date: "09-08 12:30",
+  },
+  {
+    title: "섬 넓히기 1단계",
+    isPlus: false,
+    point: 10,
+    date: "09-08 12:30",
+  },
+  {
+    title: "재활용 습관 들이기",
+    isPlus: true,
+    point: 10,
+    date: "09-08 12:30",
+  },
+];
+
+const Title = styled.a`
+  font-family: "Pr-Bold";
+  font-size: 1.1rem;
+  max-width: 9.5rem;
+  margin-right: 0.4rem;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+  cursor: pointer;
+  :hover {
+    text-decoration: underline;
+  }
+`;
+
+const CustomTooltip = style(({ className, ...props }: TooltipProps) => (
+  <TooltipLib.default {...props} arrow classes={{ popper: className }} />
+))(({ theme }) => ({
+  [`& .${tooltipClasses.arrow}`]: {
+    color: theme.palette.common.black,
+  },
+  [`& .${tooltipClasses.tooltip}`]: {
+    backgroundColor: theme.palette.common.black,
+    fontSize: 13,
+    fontFamily: "Pr-Bold",
+    paddingTop: 8,
+    paddingRight: 15,
+    paddingBottom: 8,
+    paddingLeft: 15,
+  },
+}));
+
+const insert = () => {
```

Comment:
```suggestion
const insert = (points) => {
```
해서, points라는 parameter를 두고, insert(dummyPoint)로 호출해서 나중에 값 바꿔치기 하기 좋게 만들면 좋을 것 같습니답